examples/gridfs_stream.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * This example demonstrates how to use GridFS streams.
+ */
+
+declare(strict_types=1);
+
+namespace MongoDB\Examples;
+
+use MongoDB\BSON\ObjectId;
+use MongoDB\Client;
+
+use function assert;
+use function fclose;
+use function feof;
+use function fread;
+use function fwrite;
+use function getenv;
+use function strlen;
+
+require __DIR__ . '/../vendor/autoload.php';
+
+$client = new Client(getenv('MONGODB_URI') ?: 'mongodb://127.0.0.1/');
+// Disable MD5 computation for faster uploads, this feature is deprecated
```

Comment:
I explicitly disabled MD5, that's what developers should do in their applications.